tests/integration/istioio/trafficmanagement/circuitbreaker/scripts/circuitbreaker_setup.txt

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+cat<<EOF | kubectl apply -f -
```

Comment:
Are you deploying bookinfo before these?

Reply:
No. I'm not. I did that to enable automatic injection.